Code/Framework/AzTest/AzTest/AzTest.h

```diff
@@ -193,14 +193,9 @@ namespace AZ
         {
             static_assert(std::is_base_of<BenchmarkEnvironmentBase, T>::value, "Supplied benchmark environment must be derived from BenchmarkEnvironmentBase");
 
-            static AZ::EnvironmentVariable<AZ::Test::BenchmarkEnvironmentRegistry> s_benchmarkRegistry;
-            if (!s_benchmarkRegistry)
-            {
-                s_benchmarkRegistry = AZ::Environment::CreateVariable<AZ::Test::BenchmarkEnvironmentRegistry>(s_benchmarkEnvironmentName);
-            }
-
+            static AZ::Test::BenchmarkEnvironmentRegistry s_benchmarkRegistry;
             auto benchmarkEnv{ new T };
-            s_benchmarkRegistry->AddBenchmarkEnvironment(std::unique_ptr<BenchmarkEnvironmentBase>{ benchmarkEnv });
+            s_benchmarkRegistry.AddBenchmarkEnvironment(std::unique_ptr<BenchmarkEnvironmentBase>{ benchmarkEnv });
```

Comment:
This has some behavioural changes that I don't think are intended.  The first of which is the `BenchmarkEnvironmentRegistry` env variable is no longer being created, so the calls to `FindVariable` in the `AZ_BENCHMARK_HOOK*` macros won't process registered environments.